.github/workflows/int-and-connected-test-run.yml

```diff
@@ -159,137 +91,76 @@ jobs:
+      - name: Checkout code
+        uses: actions/checkout@v4
+      - name: Set up JDK 17
+        uses: actions/setup-java@v4
+        with:
+          distribution: 'temurin'
+          java-version: '17'
+          cache: 'gradle'
+      - name: Whitelist Runner IP
         uses: broadinstitute/datarepo-actions/actions/main@0.74.0
         with:
           actions_subcommand: 'gcp_whitelist'
           sa_b64_credentials: ${{ secrets.SA_B64_CREDENTIALS }}
-      - name: "Check for an available namespace to deploy API to and set state lock"
+      - name: Check for an available namespace to deploy API to and set state lock
         uses: broadinstitute/datarepo-actions/actions/main@0.74.0
         with:
           actions_subcommand: 'k8_checknamespace'
           k8_namespaces: 'integration-1,integration-2,integration-3,integration-6'
           sa_b64_credentials: ${{ secrets.SA_B64_CREDENTIALS }}
-      - name: "Build docker container via Gradle"
+      - name: Build docker container via Gradle
         uses: broadinstitute/datarepo-actions/actions/main@0.74.0
         with:
           actions_subcommand: 'gradlebuild' # creates gcr build with git_hash tag
           sa_b64_credentials: ${{ secrets.SA_B64_CREDENTIALS }}
-      - name: "Deploy to cluster with Helm"
+      - name: Deploy to cluster with Helm
         uses: broadinstitute/datarepo-actions/actions/main@0.74.0
         with:
           actions_subcommand: 'helmdeploy'
           helm_create_secret_manager_secret_version: 0.0.8
-          helm_datarepo_api_chart_version: 0.0.739
-          helm_datarepo_ui_chart_version: 0.0.360
+          helm_datarepo_api_chart_version: 0.0.744
+          helm_datarepo_ui_chart_version: 0.0.362
           helm_gcloud_sqlproxy_chart_version: 0.19.13
           helm_oidc_proxy_chart_version: 0.0.44
           sa_b64_credentials: ${{ secrets.SA_B64_CREDENTIALS }}
-      - name: "Fetch gitHash for deployed integration version"
+      - name: Fetch gitHash for deployed integration version
         id: configuration
         run: |
           git_hash=$(git rev-parse --short HEAD)
           echo "git_hash=${git_hash}" >> $GITHUB_OUTPUT
           echo "Latest git hash for this branch: $git_hash"
-      - name: "Wait for deployment to come back online"
+      - name: Wait for deployment to come back online
         uses: broadinstitute/datarepo-actions/actions/wait-for-deployment@0.74.0
         timeout-minutes: 20
         env:
           DESIRED_GITHASH: ${{ steps.configuration.outputs.git_hash }}
           DEPLOYMENT_TYPE: 'api'
-      - name: "Run Test Runner smoke tests via Gradle"
+      - name: Run test runner smoke tests via Gradle
         uses: broadinstitute/datarepo-actions/actions/main@0.74.0
         with:
           actions_subcommand: 'gradletestrunnersmoketest'
           sa_b64_credentials: ${{ secrets.SA_B64_CREDENTIALS }}
-      - name: "Run integration tests via Gradle"
+      - name: Run integration tests via Gradle
```

Comment:
Reviewer note: per discussion with DE and TDR team this morning, the integration tests will be handled in a separate PR.